src/core_plugins/console/public/src/input.js

```diff
@@ -178,8 +178,13 @@ export function initializeInput($el, $actionsEl, $copyAsCurlEl, output) {
 
             var warnings = xhr.getResponseHeader("warning");
             if (warnings) {
-              warnings = _.map(warnings.split(", "), function (warning) {
-                  return "#! Deprecation: " + warning;
+              // pattern for valid warning header
+              var re = /\d{3} [^ ]+ \"([^"]*)\"( \"[^"]*\")/
```

Comment:
After additional discussion with @clintongormley and @bleskes, it has been agreed that we should allow escaped quotes in warning messages, so I will work on a change on both sides to support them. Thanks for raising the question @jbudz. 😄